paychmgr/paychget_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/filecoin-project/lotus/chain/actors/builtin/paych"
 	paychmock "github.com/filecoin-project/lotus/chain/actors/builtin/paych/mock"
 	"github.com/filecoin-project/lotus/chain/types"
+
```

Comment:
Unfortunately, the bundles only get registered if we pull in the build package and the build package already depends on the actors package.

I may be able to refactor some things to avoid this issue, but it's probably best to leave it up to a followup patch.

Reply:
Hm. No, that wasn't the issue. Fixed.